src/scripts/main.js

```diff
@@ -1,3 +1,7 @@
 'use strict';
 
-// write code here
+const body = document.querySelector('body .wall .spider');
+
+body.style.top = '50%';
+body.style.left = '50%';
+body.style.transform = 'translateX(-50%)translateY(-50%)';
```

Comment:
use shorthand properties if it's possible
```suggestion
body.style.transform = 'translate(-50%, -50%)';
```